src/HotChocolate/Core/src/Types/SchemaBuilder.Lazy.cs

```diff
@@ -36,6 +36,7 @@ public ISchema Schema
                     _schema = value ?? throw new ArgumentNullException(nameof(value));
                     _isSet = true;
                     Completed?.Invoke(this, EventArgs.Empty);
+                    Completed = null;
```

Comment:
@michaelstaib  this is where the definitions were kept alive

Reply:
good catch